approval_polls/templates/base.html

```diff
@@ -45,6 +45,11 @@
               </a>
             </div>
           </div>
+          <div class='row-fluid'>
```

Comment:
Again here could you add the `<p> </p>` tags to keep some space between the logo and the `View trending polls` ?